basics.js

```diff
@@ -4,19 +4,27 @@
 
 var pets = ['cat', 'dog', 'wombat'];
 
+for (var i = 0; i < pets.length; i += 1) {
+  console.log("I have a " + pets[i]);
+}
+
 ///// Question 2
 // Change my friend's hair color to "green", without modifying the line below.
 
 var friend = { name: 'Chris', age: 27, hairColor: 'blue' };
 
+friend.hairColor = "green";
+
 ///// Question 3
 // The function below is problematic. In 1-2 sentences explain why, and then fix the code.
 
 var squarePlusOne = function(n){
-  squared = n * n;
+  squared = var n * n;
```

Comment:
So close! I see your logic here, but you need to put the `var` keyword in front of squared.